core-java-modules/core-java-io/src/test/java/com/baeldung/extension/ExtensionFromMimeTypeUnitTest.java

```diff
@@ -0,0 +1,47 @@
+package com.baeldung.extension;
+
+import static org.junit.Assert.assertEquals;
+import java.util.HashMap;
+import java.util.Map;
+
+import org.apache.tika.mime.MimeType;
+import org.apache.tika.mime.MimeTypeException;
+import org.apache.tika.mime.MimeTypes;
+import org.junit.Test;
+
+import com.j256.simplemagic.ContentInfo;
+
+public class ExtensionFromMimeTypeUnitTest {
+    public static final String PNG_EXT = "image/png";
+    @Test
+    public void whenUsingTika_thenGetFileExtension() throws MimeTypeException {
+        MimeTypes allTypes = MimeTypes.getDefaultMimeTypes();
```

Comment:
Since we have two MimeTypes imports in this test, let's use the fully qualified name here too, to make it clear what the import is

Reply:
Sure, made the changes.